src/Core/Client/RetryMiddleware.php

```
@@ -0,0 +1,191 @@
+     * @return callable
+     */
+    private function onRejected(RequestInterface $req, array $options): callable
+    {
+        $retryAttempt = $options['retryAttempt'];
+        $maxRetries = $options['maxRetries'];
+        return function ($reason) use ($req, $options, $retryAttempt, $maxRetries) {
+            if (!$this->shouldRetry(
+                $retryAttempt,
+                $maxRetries,
+                null,
+                $reason
```

PHPStan (2.1.17)

Parameter #4 $exception of method Intercom\Core\Client\RetryMiddleware::shouldRetry() expects Throwable|null, mixed given.


(argument.type)